smile_id_core/web_api.py

```diff
@@ -18,14 +18,17 @@ class WebApiTest(ApiBase):
     SERVER_URL = Servers.TEST_SERVER_URL
 
     class Urls:
-        GET_JOB_STATUS = "{server_url}/job_status"
-        GET_SERVICES = "{server_url}/services"
-        UPLOAD = "{server_url}/upload"
-        VERIFY_DOCUMENT = "{server_url}/id_verification"
+        GET_JOB_STATUS = "{server_url}/v{api_version}/job_status"
+        GET_SERVICES = "{server_url}/v{api_version}/services"
+        UPLOAD = "{server_url}/v{api_version}/upload"
+        VERIFY_DOCUMENT = "{server_url}/v{api_version}/id_verification"
+        VERIFY_DOCUMENT_ASYNC = "{server_url}/v{api_version}/verify_async"
+
+    def __init__(self, partner_id: str, api_key: str, api_version: int = 1, call_back_url: str = None):
```

Comment:
It's a bad idea to add a positional parameter between other positional parameters. Or at least it's required to make sure that this constructor is not called anywhere with positional parameters, otherwise what was previously `call_back_url` could accidentally be mapped to `api_version` now.

Reply:
+1